build-tools/packages/build-cli/src/commands/generate/changeset.ts

```diff
@@ -309,21 +345,20 @@ export default class GenerateChangesetCommand extends BaseCommand<
 				// This is typed as any because the typings don't include the optionsPerPage property.
 				// eslint-disable-next-line @typescript-eslint/no-explicit-any
 			} as any,
```

Comment:
the use of any here means I can't rely on the compiler to code review that the types of this item are correct. Would be much better to work around that issue in a way that still lets the compiler check some things. Maybe make the array of `prompts.PromptObject & {optionsPerPage?: number}`

Reply:
Great idea; done in latest.